components/brave_rewards/common/pref_names.cc

```diff
@@ -13,6 +13,8 @@ const char kUserVersion[] = "brave.rewards.user_version";
 const char kCurrentUserVersion[] = "2.5";
 const char kHideButton[] = "brave.hide_brave_rewards_button";
 const char kShowButton[] = "brave.show_brave_rewards_button";
+const char kShowLocationBarButton[] =
+    "brave.rewards.show_brave_rewards_button_in_location_bar";
```

Comment:
I'm all for renaming the constant, but are we gaining anything by renaming the actual underlying pref? Sorry if I'm missing something obvious there...

Reply:
So, I am transitioning here from the current pref because it is currently overloaded: it controls both the Rewards icon in the location bar and the inline tipping buttons. Since we need to respect the current user's setting we need to set the new pref for tipping buttons tot he value of the `show` pref, but I don't think there is a way to do it correctly without creating a new pref for `show` as well and clearing the old pref during migration.

Reply:
I see, makes sense.